src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.WindowWalker/Commands/CloseWindowCommand.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CmdPal.Ext.WindowWalker.Components;
+using Microsoft.CmdPal.Ext.WindowWalker.Properties;
+using Microsoft.CmdPal.Extensions;
+using Microsoft.CmdPal.Extensions.Helpers;
+
+namespace Microsoft.CmdPal.Ext.WindowWalker.Commands;
+
+internal sealed partial class CloseWindowCommand : InvokableCommand
+{
+    private readonly Window _window;
+
+    public CloseWindowCommand(Window window)
+    {
+        Icon = new("\xE8BB");
+        Name = $"{Resources.wox_plugin_windowwalker_Close} (Ctrl+F4)";
```

Comment:
same comment about the shortcut text here

Reply:
done.